spec/cictl/index_command_spec.rb

```diff
@@ -120,13 +112,17 @@
   end
 
   describe "#index today" do
+    # Create new update and delete files in a temp directory based on fixtures.
+    after(:each) { HathiTrust::Services.register(:data_directory) { @save_dd } }
```

Comment:
The helper doesn't work here, because we need to do part of the test with the original `data_directory` and then later override it. That might point to a need to separately define `fixtures` or something for the tests rather than doing everything with `data_directory`, but I don't think it's worth trying to untangle right now.